packages/slate/src/transforms/text.ts

```diff
@@ -177,7 +177,8 @@ export const TextTransforms: TextTransforms = {
         const { path } = point
         const { offset } = start
         const text = node.text.slice(offset)
-        editor.apply({ type: 'remove_text', path, offset, text })
+        if (text.length > 0)
+          editor.apply({ type: 'remove_text', path, offset, text })
```

Comment:
Can you remove these checks in this file, since the change above makes `editor.apply` a no-op already I don't think we need to guard against it everywhere else?

Reply:
And/or, leave the checks here and remove them from editor.apply?

The benefit of having them here is that they'll save these no-op editor.apply calls from even happening, also saving any plugins that hook into editor.apply from dealing with them.  I believe my issues from this actually came from plugins not handling it well.  Arguably plugins should just check that text.length > 0 as well, but then that creates a kind-of undocumented protocol for this.  Up to you, lmk :)

Reply:
Ah interesting, that's a good point. In that case keeping them sounds both like a good idea I agree. Thanks!